src/drive/locales/en.json

```diff
@@ -733,5 +733,18 @@
         "second": "Check our plans"
       }
     }
+  },
+  "FileDivergedModal": {
+    "title": "Your version differs from the local version",
+    "content": "Someone has modified the file outside Cozy while you were editing it, you can retrieve their modifications instead of yours or continue your editing in a new file.",
```

Comment:
Are we sure about that text? Technically it's outside of the OO server (can be the one of rennes or other)

Reply:
It's mistake from me

Reply:
I'm discussing it with Zoé